avatar/test_utils.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2023 Google LLC
```

Comment:
Could you rename this file to `common.py` ?

Reply:
changed in https://github.com/google/avatar/pull/7